### ISSUE TITLE
Adding new strings in language files

### ISSUE THREAD

**Issue by patriksh** [navjottomer/osclass-languages#2]
Any ideas what software was used to create the .pot/.po files for Osclass?

[Eazy Po](http://www.eazypo.ca/) does the job for me when making plugins/themes, but Osclass has different files for flash messages, core, and theme (wth is theme)?

**Comment by navjottomer** [comment]
A single language file is good enough.

**Comment by patriksh** [comment]
> A single language file is good enough.

So we should use only one language file for future language updates?

**Comment by navjottomer** [comment]
Yup and leave the theme.po file as every theme has it's own language file.
